src/pybullet_helpers/link.py

```diff
@@ -0,0 +1,43 @@
+"""Pybullet helper class for link utilities."""
+from typing import NamedTuple
+
+import pybullet as p
+
+from predicators.src.pybullet_helpers.geometry import Pose, Pose3D, \
+    Quaternion, get_pose
+
+BASE_LINK: int = -1
+
+
+class LinkState(NamedTuple):
+    """Link state from Pybullet."""
```

Comment:
similarly to before, add the rationale for NamedTuple and mention the pybullet function that this mimics